eventing/README.md

```diff
@@ -1,64 +1,159 @@
+    yet being produced.
+1.  Services can be connected to create new applications
     - without modifying producer or consumer.
     - with the ability to select a specific subset of events from a particular
-    producer
-
-The above concerns are consistent with the [design goals](https://github.com/cloudevents/spec/blob/master/spec.md#design-goals) of CloudEvents, a common specification for cross-service interoperability being developed by the CNCF Serverless WG.
-
-## System design notes
-
-We’re following an agile process, starting from a working prototype that
-addresses a single use case and iterating to support additional use cases.
-
-Knative Events depends on [knative/serving](https://github.com/knative/serving) and
-together with [knative/build](https://github.com/knative/build) provides a
-complete serverless platform.
-
-The primary goal of *events* is interoperability; therefore, we expect to
-provide common libraries that can be used in other systems to emit or consume
-events.
-
-
-## Naming
-
-We'll be tracking the CloudEvents nomenclature as much as possible; however,
-that is in flux and many of the needed terms are outside of the current scope
-of the specification. We use the term "feed" to represent the concept of
-attaching an event (or filtered event stream via a "trigger) to an action.
-
-# Getting Started
-
-* [Setup Knative Serving](https://github.com/knative/docs/blob/master/install/README.md)
-* [Setup Knative Eventing](https://github.com/knative/eventing/blob/master/DEVELOPMENT.md)
-* [Run samples](https://github.com/knative/eventing/blob/master/sample/README.md)
-
-
-# Kubernetes Extensions
-
-Eventing extends Kubernetes with Custom Resource Definitions (CRDs). The
-following Custom Resources (CRs) are used in the production and consumption of
-events:
-
-#### Channels
-
- - Bus, ClusterBus
- - Channel
- - Subscription
-
-#### Feeds
-
- - Feed
- - EventSource
- - EventType
+      producer
+
+The above concerns are consistent with the
+[design goals](https://github.com/cloudevents/spec/blob/master/spec.md#design-goals)
+of CloudEvents, a common specification for cross-service interoperability being
+developed by the CNCF Serverless WG.
+
+> **Note**: The Eventing repo is very much a work-in-progress. See the
+> [list of Known Issues](https://github.com/knative/eventing/issues?q=is%3Aissue+is%3Aopen+label%3A%22Known+Issue%22)
+> with the current state, but feel free to report additional issues. Expect to
+> see changes in all of the following areas in the next months:
+>
+> 1.  The API model and terminology is still evolving.
+> 1.  The controller model for Buses and Sources are still being refined.
+> 1.  Test coverage is not up to snuff.
+> 1.  The release packaging is still being determined.
+
+Once these concerns are resolved, we expect to work on creating a wider array of
+EventSources.
+
+## Installation
+
+You can install the Flows definition and the core controllers for Knative
```

Comment:
After reading through the whole thing, I think it might make the most sense to rewrite this line as "You can install Knative Eventing with the following command:", to avoid mentioning Flows here.

Reply:
After reading through everything here, I think it might make the most sense to make this sentence just "You can install Knative Eventing with the following command:" to avoid introducing Flows at this point.

Reply:
Thanks; I moved this section and clearly didn't rework the text.